### ISSUE TITLE
Lack of monochromatic color contrast for clickables

### ISSUE THREAD

**Issue by kaBeech** [kaBeech/guideme#115]
Other variations in color perception still show noticeable contrast, but monochrome contrast is negligible:

![coluguide_full_color](https://github.com/user-attachments/assets/a0d01bee-45d0-43b3-96e5-2bd6a4e8efe0)
![coluguide_protanopia](https://github.com/user-attachments/assets/5d4572e9-198c-4548-ab86-cebd6cb9bc11)
![coluguide_deuteranopia](https://github.com/user-attachments/assets/5d67bd03-88c0-4466-a722-5314f15382bf)
![coluguide_tritanopia](https://github.com/user-attachments/assets/63841898-3f8e-4a0a-9feb-958ba29550cc)
![coluguide_monochrome](https://github.com/user-attachments/assets/aabf64d3-59c3-4ca8-a7b4-d23f96f1e7c0)

I previously excused this by saying that few people have full achromatopsia, but I no longer find this argument compelling.
